### PR TITLE
Make scrollbars appear in engine-specific location

### PR DESCRIPTION
**Pull request by davidsgrogan**
@dholbert, can you review? This should fix css/css-flexbox/overflow-auto-005.html and https://bugzilla.mozilla.org/show_bug.cgi?id=1703074

### CODE REVIEW

File: css/css-flexbox/overflow-auto-005.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <link rel="help" href="https://www.w3.org/TR/css-overflow-3/#propdef-overflow">
 <link rel="help" href="https://www.w3.org/TR/css-overflow-4/#classic-scrollbars">
 <link rel="match" href="reference/overflow-auto-005-ref.html">
+<meta name="fuzzy" content="0-3;0-3">
```
Comment (davidsgrogan) [inline css/css-flexbox/overflow-auto-005.html:7]: A few pixels in Firefox are off by a small amount.

**Review by dholbert — APPROVED** [review]
Looks good to me. Thanks!

**Comment by dholbert** [comment]
(I don't review WPT tests commonly enough to know if I should hit the squash-and-merge button myself, after reviewing, vs. if I should defer to someone else to do that. I'll leave it unclicked for now.)

**Comment by davidsgrogan** [comment]
I think we can hit Squash and merge. I'm about to try it and I hope someone will yell at me if I wasn't supposed to.

**Pull request merged**